crates/core/src/db/datastore/locking_tx_datastore/committed_state.rs

```diff
@@ -399,21 +399,25 @@ impl CommittedState {
                     let pv = table
                         .delete(blob_store, row_ptr, |row| row.to_product_value())
                         .expect("Delete for non-existent row!");
```

Comment:
If this panics, then metrics count in prometheus will permanently become incosistent with table but then we will also have other bigger troubles?